gradle-plugins/compose/src/main/kotlin/org/jetbrains/compose/ComposePlugin.kt

```diff
@@ -65,28 +63,34 @@ abstract class ComposePlugin : Plugin<Project> {
     @Suppress("DEPRECATION")
     class Dependencies(project: Project) {
         val desktop = DesktopDependencies
-        val animation get() = composeDependency("org.jetbrains.compose.animation:animation")
-        val animationGraphics get() = composeDependency("org.jetbrains.compose.animation:animation-graphics")
-        val foundation get() = composeDependency("org.jetbrains.compose.foundation:foundation")
-        val material get() = composeDependency("org.jetbrains.compose.material:material")
-        val material3 get() = composeDependency("org.jetbrains.compose.material3:material3")
-        val material3AdaptiveNavigationSuite get() = composeDependency("org.jetbrains.compose.material3:material3-adaptive-navigation-suite")
-        val runtime get() = composeDependency("org.jetbrains.compose.runtime:runtime")
-        val runtimeSaveable get() = composeDependency("org.jetbrains.compose.runtime:runtime-saveable")
-        val ui get() = composeDependency("org.jetbrains.compose.ui:ui")
+        val animation = composeDependency("org.jetbrains.compose.animation:animation")
+        val animationGraphics = composeDependency("org.jetbrains.compose.animation:animation-graphics")
+        val foundation = composeDependency("org.jetbrains.compose.foundation:foundation")
+        val material = composeDependency("org.jetbrains.compose.material:material")
+        val material3 = composeDependency("org.jetbrains.compose.material3:material3")
+        val material3AdaptiveNavigationSuite = composeDependency("org.jetbrains.compose.material3:material3-adaptive-navigation-suite")
+        val runtime = composeDependency("org.jetbrains.compose.runtime:runtime")
+        val runtimeSaveable = composeDependency("org.jetbrains.compose.runtime:runtime-saveable")
+        val ui = composeDependency("org.jetbrains.compose.ui:ui")
+
         @Deprecated("Use desktop.uiTestJUnit4", replaceWith = ReplaceWith("desktop.uiTestJUnit4"))
         @ExperimentalComposeLibrary
-        val uiTestJUnit4 get() = composeDependency("org.jetbrains.compose.ui:ui-test-junit4")
+        val uiTestJUnit4 = composeDependency("org.jetbrains.compose.ui:ui-test-junit4")
+
+        @ExperimentalComposeLibrary
+        val uiBackHandler = composeDependency("org.jetbrains.compose.ui:ui-backhandler")
```

Comment:
I'd prefer not adding such new API at all.
1. We're not super happy of having such aliases at all. Probably most relevant task about it is [CMP-5861](https://youtrack.jetbrains.com/issue/CMP-5861)
2. We don't have plans to stabilize this module at all. It's opposite - it will be replaced to [navigationevent](https://developer.android.com/jetpack/androidx/releases/navigationevent) soon-ish